lib/subscription-connection.js

```diff
@@ -278,13 +275,11 @@ module.exports = class SubscriptionConnection {
           return print(document)
         },
         pubsub: sc,
-        lruGatewayResolvers: this.lruGatewayResolvers,
         reply: {
           entityResolversFactory: this.entityResolversFactory,
           request: context.request,
           [kLoaders]:
-            subscriptionLoaders && subscriptionLoaders.create(context),
-          [kEntityResolvers]: this.entityResolvers
```

Comment:
This was important.

Reply:
`this.entityResolvers` seems not initialized. [kEntityResolvers] is always undefined. Unless I'm missing something

Reply:
I'm pretty sure those are not _always_ undefined.

Reply:
The `this.entityResolvers` isn't mentioned anywhere else in the `SubscriptionConnection` class, is not initialized.
Where the subscriptionConnection class is used there is no any external assignment like `connection. entityResolvers = something`

I checked on the original commit when it was added the line as well and same stuff.

I've added a log for the value in the tests, same. The `this.entityResolvers` is always undefined.